### PR TITLE
[dvs/NAT] Fix for test_VerifyConntrackTimeoutForNatEntry test case

### PR DESCRIPTION
**Pull request by AkhileshSamineni**
This PR is to address the issue https://github.com/Azure/sonic-swss/issues/1409.

Signed-off-by: Akhilesh Samineni <akhilesh.samineni@broadcom.com>

### CODE REVIEW

File: tests/test_nat.py [hunk 1]
```diff
@@ -308,9 +307,7 @@ def _check_conntrack_for_static_entry():
 
             proto_index = conntrack_list.index("udp")
 
-            # FIXME: conntrack_list[proto_index + 7] is consistently returning 431995. Need the feature owner
-            # to confirm if this check is wrong or if the behavior is wrong.
-            if int(conntrack_list[proto_index + 7]) < 432000 and int(conntrack_list[proto_index + 7]) > 431900:
+            if int(conntrack_list[proto_index + 7]) > 432000 or int(conntrack_list[proto_index + 7]) < 431900:
                 return (False, None)
```
Comment (daall) [inline tests/test_nat.py:312]: I missed this when I was refactoring, but there also needs to be a `return (True, None)` at the end if all the previous checks succeed. Oops.

Comment (AkhileshSamineni) [reply tests/test_nat.py:311]: Done.

**Comment by AkhileshSamineni** [comment]
> See comment. Also please address the LGTM warning. Thank you! :)

Fixed LGTM warning.

**Comment by AkhileshSamineni** [comment]
Retest vs please.

**Comment by AkhileshSamineni** [comment]
retest this please.

**Comment by AkhileshSamineni** [comment]
@daall  I see below 3 test cases are failed which are not related this PR changes.

test_port_dpb_vlan.TestPortDPBVlan.test_one_port_one_vlan
test_port_dpb_vlan.TestPortDPBVlan.test_one_port_multiple_vlan
test_port_dpb_vlan.TestPortDPBVlan.test_all_port_10_vlans

**Comment by daall** [comment]
retest vs please

**Comment by daall** [comment]
> @daall I see below 3 test cases are failed which are not related this PR changes.
> 
> test_port_dpb_vlan.TestPortDPBVlan.test_one_port_one_vlan
> test_port_dpb_vlan.TestPortDPBVlan.test_one_port_multiple_vlan
> test_port_dpb_vlan.TestPortDPBVlan.test_all_port_10_vlans

I'm pretty sure they're unrelated. I'll give it one more try and then go ahead and merge this. Thanks for the quick turnaround! :)

**Comment by AkhileshSamineni** [comment]
@daall  This time also test cases related to DPB got failed.

test_port_dpb.TestPortDPB.test_port_breakout_one
test_port_dpb.TestPortDPB.test_port_breakout_multiple
test_port_dpb_acl.TestPortDPBAcl.test_one_acl_table_many_ports
test_port_dpb_acl.TestPortDPBAcl.test_one_port_many_acl_tables
test_port_dpb_acl.TestPortDPBAcl.test_many_ports_many_acl_tables
test_port_dpb_vlan.TestPortDPBVlan.test_dependency
test_port_dpb_vlan.TestPortDPBVlan.test_one_port_one_vlan
test_port_dpb_vlan.TestPortDPBVlan.test_one_port_multiple_vlan
test_port_dpb_vlan.TestPortDPBVlan.test_all_port_10_vlans

Please go ahead with merge.

**Comment by arlakshm** [comment]
retest vs please

**Comment by daall** [comment]
> @daall This time also test cases related to DPB got failed.
> 
> test_port_dpb.TestPortDPB.test_port_breakout_one
> test_port_dpb.TestPortDPB.test_port_breakout_multiple
> test_port_dpb_acl.TestPortDPBAcl.test_one_acl_table_many_ports
> test_port_dpb_acl.TestPortDPBAcl.test_one_port_many_acl_tables
> test_port_dpb_acl.TestPortDPBAcl.test_many_ports_many_acl_tables
> test_port_dpb_vlan.TestPortDPBVlan.test_dependency
> test_port_dpb_vlan.TestPortDPBVlan.test_one_port_one_vlan
> test_port_dpb_vlan.TestPortDPBVlan.test_one_port_multiple_vlan
> test_port_dpb_vlan.TestPortDPBVlan.test_all_port_10_vlans
> 
> Please go ahead with merge.

Yeah there seems to be some problem with the DPB tests, I am following up with them. This is OK to merge. Thanks again!

**Pull request merged**